dev/io.openliberty.microprofile.openapi.3.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml

```diff
@@ -0,0 +1,45 @@
+<!--Copyright (c) 2018 IBM Corporation and others. All rights reserved.
+    This program and the accompanying materials are made available under the 
+    terms of the Eclipse Public License v1.0 which accompanies this distribution, 
+    and is available at 
+        http://www.eclipse.org/legal/epl-v10.html 
+    Contributors: 
+        IBM Corporation - initial API and implementation
+-->
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="microprofile-openapi-2.0-tck" verbose="2" configfailurepolicy="continue">
```

Comment:
Is this needed?

It looks like a bit of a mess of commented out lines which isn't actually used because it's not referenced in the pom.

Reply:
i'll tidy this up and use it